### PR TITLE
Use cronjob to search for failed workflows

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
## Description

Add new workflow to check failed workflows that update cmd-apps versions. It reruns them if finds any.

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionality
- [ ] Documentation
- [ ] Refactoring
- [x] CI

### CODE REVIEW

File: .github/workflows/pr-for-updates.yaml [hunk 1]
```diff
@@ -10,16 +10,13 @@ jobs:
...
-          echo "$PULL_REQUEST_BODY" >> $GITHUB_ENV
-          echo "EOF" >> $GITHUB_ENV
-      - name: pull-request-action
-        uses: vsoch/pull-request-action@add/retry
+          ref=${{ github.ref }}
+          branch=${ref#"refs/heads/"}
+
+          prTitle="Update from $branch"
+          prBody=$(git log --pretty='format:%B' -1 | sed '/^$/d;$d' | sed 's/#/# /g')
+          gh pr create -B "main" -H "$ref" --title "$prTitle" --body "$prBody" ; then
```
Comment (glazychev-art) [inline .github/workflows/pr-for-updates.yaml:20]: Is `; then` a typo?

File: .github/workflows/check-failed-workflows.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+name: Check failed auto-pull-request workflows
+on:
+  schedule:
+    # Every hour
+    - cron: '0 */1 * * *'
```
Comment (denis-tingaikin) [inline .github/workflows/check-failed-workflows.yaml:6]: per hour could be too expensive. 

Could we start with per day job?

cc @glazychev-art

**Review by denis-tingaikin — APPROVED** [review]
@glazychev-art Could you have a look?

File: .github/workflows/check-failed-workflows.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          runs=$(gh api \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          /repos/$owner/$repo/actions/runs?branch=update/$owner/${{ matrix.repository }})
+
+          lastRun=$(echo $runs | jq '[.workflow_runs | .[] | select(.name=="Pull Request on update/* Branch Push")][0]')
+          lastRunConclusion=$(echo $lastRun | jq -r '.conclusion')
+
+          echo Conclusion for ${{ matrix.repository }} is $lastRunConclusion
+          if [ "$lastRunConclusion" != "success" ]; then
```
Comment (glazychev-art) [inline .github/workflows/check-failed-workflows.yaml:43]: I have concerns.. I'm not sure if we need to handle all `!success`. What if the conclusion was `cancelled` or `skipped`?
Also, is it possible to consider only `completed` workflow_run action type?
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=completed#workflow_run

What do you think?

Comment (NikitaSkrynnik) [reply .github/workflows/check-failed-workflows.yaml:43]: I don't remember why I used `!success`. Let's try `failure` only.
I think it's possible to consider `completed` only

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**